README.md

```diff
@@ -1 +1,70 @@
-# ps_healthcheck
+# Health check
+
+## About
+
+Adds a block with several links.
+
+## Building assets
+
+If you need to change the javascript code you have to compile the assets, this operation is done
```

Comment:
Some checks are already made. You dont need to declare them again

To add your custom checks, you need to declare them as service and add a tag to identify them

The tag is admin.prestashop.healthcheck.check (?)
Your service can be declared in any services config file like the global one `app/config/services.yml`. However its better to create a file healthcheck_services.yml and import it in the global services file

The service class must implement ... 
You already have some standard checks classes ... link to laminas checks ... or you can create your own to fit your need

Here is an example to check that all yaml files in config directory are well formatted
...